webrender/src/internal_types.rs

```diff
@@ -287,6 +288,15 @@ impl PackedColor {
             a: (0.5 + color.a * COLOR_FLOAT_TO_FIXED).floor() as u8,
         }
     }
+
+    pub fn to_bgra(&self) -> PackedColor {
```

Comment:
That is the format that is used for ImageFormat::RGBA8 texture uploads. Glenn and I agreed to encapsulate this in a PackedTexel type that will at least abstract this detail a bit better rather than forcing the caller to know too much about it.